doc/config.md

```diff
@@ -210,6 +211,8 @@ Settings list:
   recommended, prefer changing tolerance for particular suites or states
   instead.
 
+* `antialiasingTolerance` — indicates minimum difference in brightness between the darkest/lightest pixel (which is adjacent to the antiasing pixel) and theirs adjacent pixels. Allows to make the algorithm for determining antialiasing less strict. By default it's 0. Not recommend to increase this value above 10. Read more about [anti-aliasing algorithm](http://www.eejournal.ktu.lt/index.php/elt/article/view/10058/5000).
```

Comment:
`antiasing pixel` – instead of `antialiasing pixel`

`Allows to make the algorithm for determining antialiasing less strict.`
you can write it shorter: 
– `Allows to make the antialiasing algorithm less strict.`

`Not recommend to increase this value above 10.`
other ways to say the same:
– `It is not recommended to set this value above 10.`
– `We recommend to keep this value below 10.`
– `It is recommended to keep this value below 10.`
– `It is recommended to keep this value between 0 and 10. By default it's 0.`

Reply:
/ok

Reply:
а нельзя эту доку унести в `looks-same`, а здесь просто отсылку туда оставить? Тогда дублировать в hermione  и здесь не придется